gnomad_qc/v3/sample_qc/create_fam.py

```diff
@@ -0,0 +1,198 @@
+from collections import defaultdict, Counter
+import argparse
+import hail as hl
+from gnomad_qc.v3.resources import (
+    meta,
+    pedigree,
+    trios,
+    get_gnomad_v3_mt,
+    ped_mendel_errors,
+    duplicates,
+    get_v3_relatedness_annotated_ht,
```

Comment:
Since we've changed gnomad_qc to be a package with submodule, running scripts now require properly setting up the environment so that gnomad_qc is in the ptyhon path. We could discuss whether another approach would be more desirable but that's a broader change. In any case,  the import is correct but requires gnomad_qc to be in the path.